docs/data-sources/eg_official_event_channels.md

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "EventGrid (EG)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_eg_official_event_channels"
+description: ""
+---
+
+# huaweicloud_eg_official_event_channels
+
+Use this data source to filter EG official event channels within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "channel_name" {}
+
+data "huaweicloud_eg_official_event_channels" "test" {
+  name = var.channel_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region where the official event channels are located.  
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the channel name used to query specified official event channel.
+
+* `enterprise_project_id` - (Optional, String) Specifies the ID of the enterprise project to which the official event
+  channels belong.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `channels` - The filtered official event channels.  
+  The [channels](#eg_official_event_channels) structure is documented below.
+
+<a name="eg_official_event_channels"></a>
+The `channels` block supports:
+
+* `id` - The ID of the official event channel.
+
+* `name` - The name of the official event channel.
+
+* `description` - The description of the official event channel.
+
+* `provider_type` - The type of the official event channel.
```

Comment:
Please list the enumeration values ​​for this field.

Reply:
all enums are explained in the description of the corresponding filter parameter